xml/operating_system_linux.xml

```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Patterns for common names of Linux operating systems.
```

Comment:
should this be renamed to be operating_system_unix ? as it covers a lot more than just linux